### ISSUE TITLE
LGPL License

### ISSUE THREAD

**Issue by Jauwaad** [simonmeaden/qhidapi#2]
Is it permissible to use this project in a closed source application using LGPL license, the source contains the lgpl-3.0.txt but there is no mention of it in the README.

**Comment by simonmeaden** [comment]
Yes as far as I'm concerned. However the project is a very old one and I haven't worked on it for several years and don't support it any more. I'll change the licensing to allow that if you want to use it.
Regards Simon Meaden

⁣Sent from BlueMail ​

**Comment by Jauwaad** [comment]
Preferably a more liberal license like BSD or MIT would be great. But LGPL
at the least.

Thank you very much. Appreciate it.

**Comment by simonmeaden** [comment]
I've re-licensed to the Zero Clause BSD license which is pretty much the most free I think. I no longer use the software and haven't even looked at it for several years and as I said don't intend to support it further. I am not even certain that it is now working. The project it was used for is now defunct and anyway under this license I can always use it myself.
